syntax = "proto3";
option java_multiple_files =  true;

option java_package="com.proto.calculator";

message SumRequest{
  int32 first_number=1;
  int32  second_number=2;
}

message SumResponse{
  int32 result=1;
}

service CalculatorService {
	rpc sum(SumRequest) returns(SumResponse);
}